### DISCUSSION TITLE
Hi, How to set my own wallpaper instead using theme based wallpaper.

### DISCUSSION THREAD

**Discussion by piyushyadav0191** [prasanthrangan/hyprdots#1783] [Q&A]
Can anyone tell me the steps to use my own wallpapers!

**Comment by kRHYME7** [comment]
@piyushyadav0191  
2 ways 

### global wallpapers mkes wallpapers available for all themes
Add ` wallAddCustomPath=  ` value inside ` ~/.config/hyde/hyde.conf `   
This explicitly tells th scripts to fetch all the wallpapers on that path
![image](https://github.com/user-attachments/assets/1a255adf-a79c-4c44-8cd7-978197a09cc9)

### Per theme wallpaper

Add the wallpapers on each theme 
each theme have there own wallpaper directory. `./wallpapers` 
![image](https://github.com/user-attachments/assets/6789cdd1-5968-43da-b778-38b6849008c0)
